### PR TITLE
bn_updagreDockerNodeTooling

### PR DESCRIPTION
**Pull request by nandotorterolo**
## Purpose
upgrade node tooling InegrationTest

## Approach
- cli : plasma-cli_2.13:0.1.1
- node: [0.1.2]

## Testing
- The docker image should be publish manually:  @admiraladmirable when you have time could you push this image for me.   docker/node/Dockerfile

### CODE REVIEW

**Comment by admiraladmirable** [comment]
@nandotorterolo The image is published: `stratalab/plasma-node-tooling:0.1.2`

File: integration/setupBifrost.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 sudo rm -fr node01
 sudo rm -fr node02
 #rm -fr staking/*
-#docker run --rm -i --user root  -p 9085:9085 -p 9084:9084 -p 9091:9091 -v (pwd)/config:/node -v (pwd)/staking:/staking:rw ghcr.io/stratalab/plasma-node:0.1.0 -- --cli true --config  /node/config.conf < config.txt
+#docker run --rm -i --user root  -p 9085:9085 -p 9084:9084 -p 9091:9091 -v (pwd)/config:/node -v (pwd)/staking:/staking:rw ghcr.io/plasmalaboratories/plasma-node:0.1.2 -- --cli true --config  /node/config.conf < config.txt
```
Comment (mundacho) [inline integration/setupBifrost.sh:6]: this commented code can maybe be removed, since we are at that

**Review by mundacho — APPROVED** [review]
LGTM

File: integration/setupBifrost.sh (hunk 1 above)
Comment (nandotorterolo) [reply integration/setupBifrost.sh:6]: will remove in next PR

**Pull request merged**